specification/deviceregistry/DeviceRegistry.Management/properties/ns_discoveredAssetsProperties.tsp

```diff
@@ -19,12 +19,25 @@ model NamespaceDiscoveredAssetProperties {
   @doc("Reference to the device that provides data for this asset. Must provide device name & endpoint on the device to use.")
   deviceRef: DeviceRef;
 
+  @doc("Human-readable display name.")
+  @added(Versions.v2025_10_01)
+  displayName?: string;
+
   @doc("URIs or type definition IDs.")
   assetTypeRefs?: string[];
 
+  @doc("Human-readable description of the asset.")
+  @added(Versions.v2025_10_01)
+  description?: string;
+
   @doc("Identifier used to detect changes in the asset.")
   discoveryId: string;
 
+  @doc("Asset ID provided by the customer.")
+  @visibility(Lifecycle.Read, Lifecycle.Create)
```

Comment:
Let's not restrict the lifecycle!  It would be better to make this just another property that can be changed or else we will run into a bunch of Arc bridge deployment errors because connectors report updates to it (connectors are essentially stateless when it comes to discover so would not know what they reported last).